publisher/config/config.go

```diff
@@ -35,13 +35,16 @@ type Config struct {
 	ArtifactsDestFolder  string // s3 mounted folder
 	ArtifactsSrcFolder   string
 	AptlyFolder          string
+	SchemaURL            string
+	Schema               string
```

Comment:
Added to the config so we can show this information in the release markersr.

Reply:
Right. They were used only to process them, download, and get the local path of the schema.